base_classes/NXdata.nxdl.xml

```diff
@@ -306,24 +306,13 @@
 	</field>
 	<field name="VARIABLE_errors" type="NX_NUMBER" nameType="any">
```

Comment:
Replacing "VARIABLE" with "FIELDNAME" would be more general and easier to understand.